lldb/source/Plugins/Platform/MacOSX/PlatformDarwin.h

```diff
@@ -187,6 +187,9 @@ class PlatformDarwin : public PlatformPOSIX {
                                              std::vector<std::string> &options,
                                              XcodeSDK::Type sdk_type);
 
+  static llvm::Expected<FileSpec>
+  ResolveSDKPathFromDebugInfo(lldb_private::Target *target);
```

Comment:
We don't need this to be in the header. We can just make it local to the `.cpp` file